src/adapters/Utils.js

```diff
@@ -0,0 +1,83 @@
+import { REACT013 } from '../version';
+
+export function mapNativeEventNames(event) {
```

Comment:
this seems like something that has to live in each adapter, and not something that can be shared in "utils"